src/atom/currentChatScreenAtom.ts

```diff
@@ -0,0 +1,10 @@
+import {atom} from 'recoil';
+
+type CurrentChatScreenType = 'SignedChatScreen' | 'AnonymousChatScreen';
+
+const currentChatScreenAtom = atom<CurrentChatScreenType | null>({
+  key: 'currentChatScreenAtom',
+  default: null
+});
+
+export default currentChatScreenAtom;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The implementation of the `currentChatScreenAtom` looks good. It's a simple Recoil atom that holds the current chat screen type, which can be either 'SignedChatScreen', 'AnonymousChatScreen', or null. This is a good use of state management to control the current chat screen state in your application.
```diff
+ import {atom} from 'recoil';
+ 
+ type CurrentChatScreenType = 'SignedChatScreen' | 'AnonymousChatScreen';
+ 
+ const currentChatScreenAtom = atom<CurrentChatScreenType | null>({
+   key: 'currentChatScreenAtom',
+   default: null
+ });
+ 
+ export default currentChatScreenAtom;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->